amxmodx/CTask.cpp

```diff
@@ -269,8 +269,10 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	for(auto i = m_Tasks.length() - 1; i != 0; i--)
```

Comment:
Yes, there are plugins relying on that order, we should always keep old behavior anyway.